app/code/Magento/CacheInvalidate/Model/PurgeCache.php

```diff
@@ -82,24 +89,24 @@ public function sendPurgeRequest($tagsPattern)
     }
 
     /**
-     * Split tags by batches
+     * Split tags into batches to suit Varnish max. header size
      *
-     * @param string $tagsPattern
+     * @param array $tags
      * @return \Generator
      */
-    private function splitTags($tagsPattern)
+    private function chunkTags($tags)
```

Comment:
Looks like we can define argument and return types here:
```suggestion
    private function chunkTags(array $tags): \Generator
```

Reply:
Done (and also on the other private method) - thanks!